pygenometracks/tracks/BedGraphMatrixTrack.py

```diff
@@ -68,6 +70,23 @@ class BedGraphMatrixTrack(BedGraphTrack):
     INTEGER_PROPERTIES = {}
     # The color cannot be set for the moment
 
+    def __init__(self, properties_dict):
+        GenomeTrack.__init__(self, properties_dict)
+
+        self.tbx = None
```

Comment:
can this move up one level in the class hierarchy as a optional function?

Reply:
Thanks good idea